lib/tasks/data_migrations/create_healthcheck_endpoints_settings.rake

```diff
@@ -0,0 +1,37 @@
+namespace :data_migrations do
+  desc 'Seeds API, Tara & SMS healthcheck endpoints'
+  task create_healthcheck_endpoint_settings: :environment do
+
+    hash = {
+      check_api_url: {
+        description: <<~TEXT.squish,
+          URL to our own auction API endpoint for health checking.
+          Must be absolute, default is http://lvh.me:3000/auctions.json.
+        TEXT
+        value: 'http://lvh.me:3000/auctions.json',
```

Comment:
Why not just `localhost? Or what's the point of using `lhv.me` in this case? Is it really on port 3000?

Reply:
Shame on me. 'll move it to just localhost.